mkosi/resources/man/mkosi.1.md

```diff
@@ -2904,6 +2904,31 @@ images you want to build.
 
 Note that the minimum required Python version is 3.9.
 
+mkosi needs unrestricted abilities to create and act within namespaces. Some
+distros restrict creation of, or capabilities within, user namespaces, which
+breaks mkosi. For information about Ubuntu and AppArmor implementations of this
+restriction, see
+https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces.
+For other systems, try researching the `kernel.unprivileged_userns_clone` or
+`user.max.user_namespace` sysctls.
+
+For Ubuntu/AppArmor systems, you should be able to remove the restrictions by
+adapting this snippet to point to your mkosi binary, copying it to
+/etc/apparmor.d/mkosi, and then running `systemctl reload apparmor`:
```

Comment:
```suggestion
`/etc/apparmor.d/mkosi`, and then running `systemctl reload apparmor`:
```

Reply:
[Seems you are right](https://ubuntu.com/server/docs/apparmor#profiles), only of a minority of the ones on my own Ubuntu system follow that style, dunno if that's because they are special or just from before the style emerged or what. Anyway, will do what the docs say, thanks.